param-docs/parameter-pages/MQTT/NUMBER.DomoticzIDX.md

```diff
@@ -0,0 +1,4 @@
+# Parameter `<NUMBER>.DomoticzIDX`
+Default Value: ``
```

Comment:
Or should it be zero?

Reply:
Actually there is no default value for the Idx in Domoticz. Either ''(null string) or zero are invalid Idx values in Domoticz, A valid Idx for a Domoticz device, is an integer greater than zero. I have changed it to zero just to indicate that a numeric value is expected. 
Thanks